searx/engines/dailymotion.py

```diff
@@ -54,45 +106,61 @@ def response(resp):
 
     # parse results
     for res in search_res['list']:
+
         title = res['title']
         url = res['url']
-        content = html_to_text(res['description'])
-        thumbnail = res['thumbnail_360_url']
+        video_id = res['id']
+
+        content = res['description']
+        if content:
+            content = content.replace('\\r', '')
+            document = lxml.html.document_fromstring(content)
+            content = " ".join(lxml.etree.XPath("//text()")(document))
```

Comment:
Why not use `searx.utils.html_to_text` ?

Reply:
I have seen issues with `&lt;` and similar HTML notations.

Reply:
I can't reproduce the bug: `searx.utils.html_to_text('<span>test &lt; </span>') == 'test <'`.

It would be better to fix the `html_to_text` function rather than introducing a new implementation here.

Reply:
Strange .. can't reproduce the `&lt;` issue nor the issue with `\\r` .. will switch back to `html_to_text(..)` / thanks!